openedx/core/djangoapps/call_stack_manager/tests/tests.py

```diff
@@ -0,0 +1,162 @@
+"""
+Test cases for Call Stack Manager
+"""
+from .models import ModelMixin, ModelNothing, ModelMixinCSM, ModelAnotherCSM
+from django.test import TestCase
+from testfixtures import LogCapture
+from openedx.core.djangoapps.call_stack_manager import donottrack
+
+
+class TestingCallStackManager(TestCase):
+    """
+    Tests for call_stack_manager package
+    """
+    def setUp(self):
+        super(TestingCallStackManager, self).setUp()
```

Comment:
You don't need to make a method to just call `super`. You can just leave the method out entirely.